lib/taskwarrior-web/services/runner.rb

```diff
@@ -1,8 +0,0 @@
-module TaskwarriorWeb::Runner
```

Comment:
I was attempting to revert some changes I made and I accidentally removed it from git. I'm sure taskwarrior-web wont work without this file, so I'll add it back in while I work on the rest of your feedback.